.bettercodehub.yml

```diff
@@ -4,3 +4,7 @@ languages:
 exclude:
 - /vendor/.*
 - /pkg/client/.*
+test:
+    include:
+      - .*/test/.*\.go
+      - /tests/bdd/bdd_test.go
```

Comment:
For including the test file can you add the component depth also
Ref: https://bettercodehub.com/docs/configuration-manual

Reply:
component_depth is already present there